source/Octopus.Tentacle.Tests.Integration/Support/PollingTentacleBuilder.cs

```diff
@@ -3,6 +3,8 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Autofac;
+using Nito.AsyncEx;
+using Nito.AsyncEx.Interop;
```

Comment:
These look like an accidental addition, though I may be be mistaken.

Reply:
The second is needed but not the first. Will clean up